### PR TITLE
Add transaction history 

### PR DESCRIPTION
**Pull request by rs-Nenad**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -61,10 +64,18 @@ export default function App() {
 
     if (lang) {
       strings.setLanguage(lang)
+      moment.locale(lang === "zh"? "zh-cn" : lang)
+      Moment.globalLocale = lang === "zh"? "zh-cn" : lang;
+      Moment.globalLocal = true;
+      console.log(moment(1316116057189).fromNow());
```
[the remark this answers is not in the document]
Comment (rs-Nenad) [reply src/App.js:70]: removing it right now

File: src/App.js [hunk 2]
```diff
@@ -73,6 +78,10 @@ export default function App() {
       return
     strings.setLanguage(newLang)
     window.localStorage.setItem('lang', newLang)
+    moment.locale(newLang === "zh"? "zh-cn" : newLang)
+    Moment.globalLocale = newLang === "zh"? "zh-cn" : newLang;
+    Moment.globalLocal = true;
+    console.log(moment(1316116057189).fromNow());
```
Comment (dragvs) [inline src/App.js:84]: @rs-Nenad will you update this part as well?

**Review by dragvs — CHANGES_REQUESTED** [review]

File: src/components/TransactionHistory.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+function getType(tx, pubkey) {
+  if (isTransfer(tx)) {
+    return _isOutgoing(tx, pubkey) ? strings.send : strings.receive;
+  }
+  return strings.other;
+}
+
+async function loadTransactions(connection, pubkey, limit) {
+  let transactions = await connection.getConfirmedSignaturesForAddress2(pubkey, {limit: limit});
```
Comment (sviatoslav-alekseev) [inline src/components/TransactionHistory.js:111]:
```suggestion
  let transactions = await connection.getConfirmedSignaturesForAddress2(pubkey, {limit});
```

File: src/components/TransactionHistory.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+  for (let i = 0; i < transactions.length; i++) {
+    const tx = await connection.getParsedConfirmedTransaction(transactions[i].signature)
```
Comment (sviatoslav-alekseev) [inline src/components/TransactionHistory.js:114]: We could avoid awaits in the loop and do one await via `Promise.all()`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Comment (dragvs) [reply src/components/TransactionHistory.js:114]: I'd skip it here

File: src/App.js (hunk 2 above)
Comment (dragvs) [inline src/App.js:84]:
```suggestion
    moment.locale(newLang === "zh"? "zh-cn" : newLang)
```

**Review by dragvs — CHANGES_REQUESTED** [review]

**Review by dragvs — APPROVED** [review]

**Pull request merged**